tests/test_readme.py

```diff
@@ -0,0 +1,59 @@
+import pathlib
+import re
+import subprocess
+
+import pytest
+
+readme_path = pathlib.Path(__file__).parent.parent / 'README.md'
```

Comment:
No none of the tests will, since the `tests` directory does not get bundled into our python package. `pathlib.Path(__file__)` will work when packaged, although .parent.parent won't navigate to a directory with `README.md`.

Reply:
>  seems safe to assume this code would only be run within the manubot repo directory structure

Yes, this should be the assumption for everything in `tests`